pepys_import/core/store/data_store.py

```diff
@@ -1419,3 +1419,10 @@ def is_schema_created(self):
             print(f"Database tables are not found! (Hint: Did you initialise the DataStore?)")
             return False
         return True
+
+    def is_empty(self):
+        """Returns True if all tables are empty, False otherwise."""
```

Comment:
I think the above comment is a little ambitious :-D

We're not actually testing all tables.  Suggest something like:
`Returns True if sample table (Privacy) is empty, False otherwise`

Reply:
Yes, you're right. I was checking sample tables from measurement and metadata tables at the beginning. That's why it's written like that 🙂 I'm changing it.

Reply:
Done:
https://github.com/debrief/pepys-import/pull/296/commits/ffded6ce619b0cefa1671471b641785425187d28